src/main/java/com/softserve/academy/spaced/repetition/service/AccountVerificationByEmailService.java

```diff
@@ -32,4 +35,11 @@ public void accountVerification(String token) {
         }
         accountRepository.save(editedAcc);
     }
+
+    public void verificationTokenForResetPassword(String token) {
+        String identifier = jwtTokenForResetPassword.decryptToken(token);
```

Comment:
Please take into account that there was a comment durring last demo. It is preferable to this method to return boolean instead of trhrowing exception.